ce/amf/contact.ts

```diff
@@ -17,6 +17,8 @@ export class Contact extends Entity implements IContact {
   /** @internal */
   override *validate(): Iterable<ValidationError> {
     yield* super.validate();
+    yield* this.validateChildKeys(['email', 'role']);
```

Comment:
What does the yield* keyword do?

Reply:
It yields the items from the call. (ie: `for( const each of super.validate() { yield each }` ) -- but more efficient.